### ISSUE TITLE
Potential out-of-date instruction when working with [vscode-hello-go]

### ISSUE THREAD

**Issue by ls-jingbo-jin** [ev3dev/vscode-ev3dev-browser#122]
**Describe the bug**
Potential out-of-date instruction when working with [vscode-hello-go](https://github.com/ev3dev/vscode-hello-go).
Step "Press F5 to download the program and run it" doesn't work.

Getting error: 
1. No files selected for download.
2. go -o hello flag provided but not defined: -o

**To Reproduce**
Following the steps described [here](https://github.com/ev3dev/vscode-hello-go/blob/master/README.md). 

**Expected behavior**
It should be able to upload to EV3 and execute the demo program successfully. 

**Screenshots**
![error 1](https://github.com/user-attachments/assets/49e05894-1b40-47cb-818f-9ca59f9932a5)
![error](https://github.com/user-attachments/assets/f0fa38e3-f7d8-4ad6-ac6c-f02cf7ea82f5)

https://github.com/user-attachments/assets/46915b83-39cb-4192-b8c2-f368d9e505c9

**Desktop (please complete the following information):**
- MacOs Sonoma Version 14.6.1
- Vs code: 1.95.3 
- Go:  go1.23.3 darwin/arm64
- EV3DEV:  ev3dev-stretch 
- EV3 wifi connection mode, can successfully ssh and connect with vscode-ev3dev-browser

PS: I am a GO learner, I made sure my current VS code setup can run go successfully, like [this one](https://www.youtube.com/watch?v=1MXIGYrMk80). 
Files under `.vscode` are unchanged, eg: using default launch.json and settings.json. 

**Additional context**
I recently came across this fascinating extension project. The functionality enabling EV3 owners to code, run, and debug in a live IDE environment is truly exciting.
I understand that neither [vscode-hello-go](https://github.com/ev3dev/vscode-hello-go) nor this project is under active development, but I’m eager to test it out nonetheless. 
My current suspicions are: 1. The Go version might not be supported. 2. VS code security setting.  
Any assistance or guidance would be greatly appreciated 🙏!

**Comment by dlech** [comment]
Does it work if you change the command to `go build -o hello`?

**Comment by ls-jingbo-jin** [comment]
> Does it work if you change the command to `go build -o hello`?

It works now and successfully got complied:
<img width="177" alt="Screenshot 2025-01-13 at 23 28 58" src="https://github.com/user-attachments/assets/8c1c03de-056f-46d2-a844-71c623f887eb" />

However the `F5` press to deploy still ends up error:
```
 *  Executing task: go -o hello 

flag provided but not defined: -o
Go is a tool for managing Go source code.
```
<img width="957" alt="Screenshot 2025-01-13 at 23 30 08" src="https://github.com/user-attachments/assets/cb1b092f-bf53-42f9-b11e-b0a5473366d7" />

**Comment by dlech** [comment]
Did you run the command manually or did you modify `.vscode/tasks.json`?

**Comment by ls-jingbo-jin** [comment]
> Did you run the command manually or did you modify `.vscode/tasks.json`?

I ran the command manually. 

After your comment, I also tried editing the original `.vscode/tasks.json` to below:
```
 "args": [
        "build", "-o", "hello"
 ]
```

Which fixed the compiling issue, should I create a PR to apply this?

However the uploading to ev3 part is still not working, despite I updated launch.json to an existing folder already:
![Screenshot 2025-01-14 at 17 16 08](https://github.com/user-attachments/assets/cc42068b-d74f-40d7-98ce-14acabb7437f)

**Comment by dlech** [comment]
I don't think `"program"` can be an absolute path. It is trying to use the path from your computer on the EV3, which is why it says it doesn't exist.

**Comment by dlech** [comment]
I don't think you should change `${workspaceFolderBasename}` to `test2`. I don't see any folder with that name, so not surprising that it says it doesn't exist.

**Comment by ls-jingbo-jin** [comment]
> I don't think you should change `${workspaceFolderBasename}` to `test2`. I don't see any folder with that name, so not surprising that it says it doesn't exist.

My understanding is this file path is the ev3 file path where we want to upload the program to. Is that not? 

![file path](https://github.com/user-attachments/assets/93016098-eeb8-430e-ae3a-c69fcfee8b4c)

If I leave it unchanged, keep using `${workspaceFolderBasename}`, it also doesn't work.
